Update collect-heroku-logs.md (#18462)
DataDog/documentation: content/en/logs/guide/collect-heroku-logs.md

* Replace `<DD_API_KEY>` with your [Datadog API Key][2].
* Replace `<ENV>` with your application's [environment][3].
* Replace `<APPLICATION_NAME>` and `<SERVICE>` with your application name.
* Replace `<HOST>` with the desired hostname.  
**Notes**:  

>    - Per the [host section][4], metrics and traces set the default hostname to the dyno name. It is not yet possible to dynamically set the dyno name as the hostname for logs. For now, to correlate between metrics, traces, and logs the `dyno` and `dynotype` tags can be used.  

Reviewer: ```suggestion
   - Per the [host section][4], metrics and traces set the default hostname to the dyno name. It is not possible to dynamically set the dyno name as the hostname for logs. Use the `dyno` and `dynotype` tags to correlate between metrics, traces, and logs.
```
